layouts/default.vue

```diff
@@ -147,14 +147,19 @@ export default Vue.extend({
     }
   },
   beforeMount() {
+    this.$nuxt.$loading.start();
     this.$store.dispatch('userViews/refresh');
+  },
+  mounted() {
+    this.$nuxt.$loading.finish();
   }
 });
 </script>
 
 <style lang="scss" scoped>
 .v-application {
   background-color: var(--v-background-base) !important;
+  margin-top: -4px;
```

Comment:
Won't this cutoff part of the header app when the loading bar isn't being shown?

Reply:
@mrtimscampi No, because I keep the component always at top (but hidden) so page doesn't jump when the progressbar disappears. If I don't do this, it's plain horrible.